data/data/powervs/loadbalancer/alb.tf

```diff
@@ -10,6 +10,15 @@ resource "ibm_is_lb" "load_balancer" {
   subnets         = [var.vpc_subnet_id]
   security_groups = [ibm_is_security_group.ocp_security_group.id]
   tags            = [var.cluster_id]
+  type            = "public"
+}
+
+resource "ibm_is_lb" "load_balancer_int" {
+  name            = "${var.cluster_id}-loadbalancer-int"
+  subnets         = [var.vpc_subnet_id]
+  security_groups = [ibm_is_security_group.ocp_security_group.id]
+  tags            = [var.cluster_id]
```

Comment:
the other providers put more than just the cluster_id https://github.com/openshift/installer/blob/65c320503d99a3eddffca3c35d74722b4f7c7ef1/data/data/aws/vpc/master-elb.tf#L10

so something like clsuter_id-lb-int

Reply:
can we have both as of now, will use whatever feasible while dealing with these resources.

Reply:
Done